models/public/vehicle-reid-0001/model.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  This model was created for the Vehicle Re-identification problem.
+  It's based on the OSNet architecture with IN blocks.
```

Comment:
You can use `ci/documentation_updater/documentation_updater.py` script to generate description from documentation.

Reply:
Fixed